File: clinical/src/test/java/com/hartwig/actin/clinical/feed/medication/MedicationEntryCreatorTest.java
package com.hartwig.actin.clinical.feed.medication;

import static org.junit.Assert.assertFalse;
import static org.junit.Assert.assertNull;
import static org.junit.Assert.assertTrue;

import org.apache.logging.log4j.util.Strings;
import org.junit.Test;

public class MedicationEntryCreatorTest {

    @Test
    public void canInterpretActiveField() {
        assertNull(MedicationEntryCreator.isActive("NULL"));
        assertTrue(MedicationEntryCreator.isActive("active"));
        assertFalse(MedicationEntryCreator.isActive(Strings.EMPTY));
    }
}

File: clinical/src/test/kotlin/com/hartwig/actin/clinical/feed/medication/MedicationEntryCreatorTest.kt
package com.hartwig.actin.clinical.feed.medication

import com.hartwig.actin.clinical.feed.medication.MedicationEntryCreator.Companion.isActive
import org.apache.logging.log4j.util.Strings
import org.assertj.core.api.Assertions.assertThat
import org.junit.Test

class MedicationEntryCreatorTest {
    @Test
    fun shouldInterpretIfMedicationIsActive() {
        assertThat(isActive("stopped")!!).isFalse()
        assertThat(isActive("active")!!).isTrue()
        assertThat(isActive(Strings.EMPTY)).isNull()
    }
}